core/storage/question/gae_models.py

```diff
@@ -42,11 +44,8 @@ class QuestionModel(base_models.VersionedModel):
     SNAPSHOT_CONTENT_CLASS = QuestionSnapshotContentModel
     ALLOW_REVERT = True
 
-    # A dict representing the question data.
-    question_data = ndb.JsonProperty(indexed=False)
-    # The schema version for the data.
-    question_data_schema_version = (
-        ndb.IntegerProperty(required=True, indexed=True))
+    # An object representing the question state data.
+    question_state_data = ndb.JsonProperty(indexed=False)
```

Comment:
Don't we need a state schema version for this? The "current state schema version" can be shared with explorations, but each datastore model instance need to know what version its dict is formatted in.